Update topics properties to match current implementation (#3139)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/topics/index.md

The Topics explainer proposes that each topic object in the array returned by
`document.browsingTopics()` would have three properties:

-  `configVersion`: a string identifying the current configuration
-  `modelVersion`: a string identifying the machine-learning classifier used to infer site
-  `taxonomyVersion`: a string identifying the set of topics currently in use by the browser

> -  `topic`: a number identifying the topic in the [taxonomy](#how-would-topics-be-curated-and-selected)

Reviewer: ```suggestion
-  `topic`: an integer identifying the topic in the [taxonomy](#how-would-topics-be-curated-and-selected)
```

Reviewer: I assume that it must be a whole number? If so, please make this update!

Author: Good to be specific — it is indeed a (positive :) integer.
